pkg/apis/extensions/v1beta1/types.go

```diff
@@ -568,6 +568,10 @@ type DaemonSetStatus struct {
 	// NumberReady is the number of nodes that should be running the daemon pod and have one
 	// or more of the daemon pod running and ready.
 	NumberReady int32 `json:"numberReady" protobuf:"varint,4,opt,name=numberReady"`
+
+	// ObservedGeneration is the most recent generation observed by the daemon set controller.
```

Comment:
All of these comments should reference the json field names rather than the Go field names.